bundles/org.openhab.binding.unifiedremote/README.md

```diff
@@ -0,0 +1,30 @@
+# UnifiedRemote Binding
+
+This binding integrates the [Unified Remote Server](https://www.unifiedremote.com/).
+
+<b>Known Limitations: It needs the web interface to be enabled on the server settings to work.</b>
+
+## Discovery
+
+Discovery works on the default discovery UDP port 9511.
+
+## Thing Configuration
+
+The Unified Remote Server Thing requires the host to be correctly configured in order to work correctly.
+Other properties like tcpPort and udpPort are not used in the initial implementation.
+
+```
+Thing unifiedremote:server:xx-xx-xx-xx-xx-xx [ host="192.168.1.10" ]
```

Comment:
This example should be moved to the end to the chapter "Full Example". You could also add an example of the .items file and the .sitemap file (which are more important that the .things file).